### PR TITLE
Translated unused entities, device registry page and domain toggler dialog

### PR DESCRIPTION
**Pull request by springstan**
Issue affected: #3429

Translated:
- device registry page
- unused entities page
- domain toggler dialog

Results:
device_areas
unused_entities

### CODE REVIEW

File: src/translations/en.json [hunk 1]
```diff
@@ -1394,7 +1398,15 @@
...
           "configure_ui": "Configure UI",
-          "unused_entities": "Unused entities",
+          "unused_entities": {
+            "title": "Unused entities",
+            "available_entities": "These are the entities that you have available, but are not in your Lovelace UI yet.",
+            "select_to_add": "Select the entities you want to add to a card and then click the add card button.",
+            "entity": "Entity",
+            "entity_id": "Entity ID",
+            "domain": "Domain",
+            "last_changed": "Last Changed"
```
Comment (bramkragten) [inline src/translations/en.json]: Move this out of `menu` and in it's own section

File: src/dialogs/device-registry-detail/dialog-device-registry-detail.ts [hunk 2]
```diff
@@ -90,7 +93,12 @@ class DialogDeviceRegistryDetail extends LitElement {
               .disabled=${this._submitting}
             ></paper-input>
             <div class="area">
-              <paper-dropdown-menu label="Area" class="flex">
+              <paper-dropdown-menu
+                label="${this.hass.localize(
+                  "ui.panel.config.integrations.config_entry.area_picker_label"
```
Comment (bramkragten) [inline src/dialogs/device-registry-detail/dialog-device-registry-detail.ts]:
```suggestion
                  "ui.panel.config.devices.area_picker_label"
```

File: src/dialogs/device-registry-detail/dialog-device-registry-detail.ts [hunk 3]
```diff
@@ -163,7 +171,11 @@ class DialogDeviceRegistryDetail extends LitElement {
       });
       this._params = undefined;
     } catch (err) {
-      this._error = err.message || "Unknown error";
+      this._error =
+        err.message ||
+        this.hass.localize(
+          "ui.panel.config.entity_registry.editor.unknown_error"
```
Comment (bramkragten) [inline src/dialogs/device-registry-detail/dialog-device-registry-detail.ts]:
```suggestion
          "ui.panel.config.devices.unknown_error"
```

File: src/dialogs/domain-toggler/dialog-domain-toggler.ts [hunk 4]
```diff
@@ -37,7 +37,11 @@ class DomainTogglerDialog extends LitElement {
         opened
         @opened-changed=${this._openedChanged}
       >
-        <h2>Toggle Domains</h2>
+        <h2>
+          ${this.hass.localize(
+            "ui.panel.config.entity_registry.toggle_domains"
```
Comment (bramkragten) [inline src/dialogs/domain-toggler/dialog-domain-toggler.ts]: This is not the `entity_registry` make a section like `ui.dialogs.domain-toggler`

File: src/dialogs/device-registry-detail/dialog-device-registry-detail.ts (hunk 3 above)
Comment (springstan) [reply src/dialogs/device-registry-detail/dialog-device-registry-detail.ts]: Implemented in https://github.com/home-assistant/home-assistant-polymer/pull/4141/commits/96f6a843a110bbac601076e246129eb7376adc0b

File: src/dialogs/device-registry-detail/dialog-device-registry-detail.ts (hunk 2 above)
Comment (springstan) [reply src/dialogs/device-registry-detail/dialog-device-registry-detail.ts]: Implemented in https://github.com/home-assistant/home-assistant-polymer/pull/4141/commits/96f6a843a110bbac601076e246129eb7376adc0b

File: src/dialogs/domain-toggler/dialog-domain-toggler.ts (hunk 4 above)
Comment (springstan) [reply src/dialogs/domain-toggler/dialog-domain-toggler.ts]: Done with https://github.com/home-assistant/home-assistant-polymer/pull/4141/commits/4a400b8fcaa3998bdb2a4c8b0350b963d124fa64

File: src/translations/en.json (hunk 1 above)
Comment (springstan) [reply src/translations/en.json]: Done with https://github.com/home-assistant/home-assistant-polymer/pull/4141/commits/4a400b8fcaa3998bdb2a4c8b0350b963d124fa64

**Review by bramkragten — APPROVED** [review]

**Pull request merged**